src/app/components/dashboard/dashboard.service.js

```diff
@@ -214,6 +214,8 @@
                             $rootScope.properties.hostname = 'Hostname not available.';
                             $log.error('Error fetching hostname.');
                         });
+              
+					PMAPIService.getMetricsMetadata(data);
```

Comment:
Yeah, right now I have pulled all the metadata into rootscope, I will clean it up as per requirement.